pysmt/smtlib/parser/parser.py

```diff
@@ -1181,6 +1239,34 @@ def _cmd_get_value(self, current, tokens):
         self.consume_closing(tokens, current)
         return SmtLibCommand(current, params)
 
+    def _cmd_get_objectives(self, current, tokens):
+        """(get-objective)"""
+        self.parse_atoms(tokens, current, 0)
+        return SmtLibCommand(current, [])
+
+    def _cmd_objective(self, current, tokens):
+        """(maximize | minimize <term> [:id <string>] [:signed] [:lower <const_term>] [:upper <const_term>])"""
```

Comment:
The `:upper`, `:lower` bits only apply to `minmax` and `maxmin`. I think this command is not considering those, no?
http://optimathsat.disi.unitn.it/pages/smt2reference.html

Reply:
`:upper` and `:lower` are not used anymore also in `minmax` and `maxmin`. Currently I haven't implement the parsing for `minmax` and `maximin` yet.